### PR TITLE
Add FLOW_SCHEDULE event to trace module

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
CC @lvscup

**Comment by fridex** [comment]
Based on discussion in https://github.com/selinon/selinon/issues/142

File: selinon/trace.py [hunk 1]
```diff
@@ -13,6 +13,10 @@
 +----------------------------+-------------------------------------+-----------------+------------------------------------+
 | Event                      |  Event description                  | Emitter         |  msg_dict.keys()                   |
 +============================+=====================================+=================+====================================+
+|                            | A flow has been scheduled.          |                 |  dispatcher_id, elective,          |
```
Comment (lvscup) [inline selinon/trace.py]: typo elective->selective

Comment (fridex) [reply selinon/trace.py]: Awesome! Thanks for the review. If everything's fine now, I will fix unrelated pylint issues and proceed with this one.

**Pull request merged**